owncloudData/src/test/java/com/owncloud/android/data/files/datasources/implementation/OCLocalFileDataSourceTest.kt

```diff
@@ -112,6 +112,17 @@ class OCLocalFileDataSourceTest {
         verify(exactly = 1) { fileDao.getFileByIdAsFlow(OC_FILE_ENTITY.id) }
     }
 
+    @Test
+    fun `getFilesForAccount returns a list of OCFile`() {
```

Comment:
You'll need to adapt the name of this test when changing the name of the method as well

Reply:
Adapted